.github/workflows/test.yml

```diff
@@ -25,9 +25,10 @@ jobs:
       - name: Setup test execution environment.
         run: pip3 install -r requirements-meta.txt
       - name: Run tox tests
-        run: tox -e test -- --durations=0 --timeout=30
+        run: tox -- --durations=0 --timeout=30
+
   typecheck:
-    runs-on: ubuntu-20.04
+    runs-on: windows-latest
```

Comment:
Personal preference but if these could stick to running on ubuntu that'd be grand :)

Reply:
The issue I had when testing this on my fork was that it threw up type errors with chrome.py as mypy cannot properly resolve the references for the winreg module. That's why I changed it to windows-latest. Ubuntu is quicker on GitHub Actions, and perhaps preferable for many other reasons, too, but we'd need to find a solution for the code in chrome.py:_find_chrome_win() first (unless we want to just `#type: ignore` that whole function)..

Reply:
Ah, hmm, ok 🤔 I will have a bit of a think, but let's keep this for now then.